docs/DATA_FORMATS_INPUT.md

```diff
@@ -2,6 +2,17 @@
 
 Telegraf is able to parse the following input data formats into metrics:
 
+<<<<<<< HEAD
```

Comment:
This file needs fixed up due to merge issues.  I just merged your updates to the JSON parser so you may need to update again too.

Reply:
im not sure i correctly followed your new format for the INPUT_DATA_FORMATS file when i resolved the merge conflict if you could take a look at that.  I think the csv is missing the proper link to its section